tests/horizontal-slider/data/references.csv

```diff
@@ -1,12 +1,13 @@
-refId,value
-author,Isabel Del Castillo
-title,Color Viewer Slider
-authorEmail,isa.delcastillo5@gmail.com
-reference,reference/2022-4-11_11511/slider-color-viewer.html
-designPattern,https://w3c.github.io/aria-practices/#slider
-example,https://w3c.github.io/aria-practices/examples/slider/slider-color-viewer.html
-slider,https://w3c.github.io/aria/#slider
-aria-labelledby,https://w3c.github.io/aria/#aria-labelledby
-aria-valuenow,https://w3c.github.io/aria/#aria-valuenow
-aria-valuemin,https://w3c.github.io/aria/#aria-valuemin
-aria-valuemax,https://w3c.github.io/aria/#aria-valuemax
\ No newline at end of file
+refId,type,value,linkText
+title,metadata,Color Viewer Slider,
+author,metadata,Isabel Del Castillo,
+authorEmail,metadata,isa.delcastillo5@gmail.com,
+reference,metadata,reference/2022-4-11_11511/slider-color-viewer.html,Color Viewer Slider - ARIA-AT Test Case
+example,metadata,https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-color-viewer/,APG Color Viewer Slider Example
+designPattern,metadata,https://www.w3.org/WAI/ARIA/apg/patterns/slider/,APG Slider Pattern
+developmentDocumentation,metadata,https://github.com/w3c/aria-at/issues/403,Test Plan Development Documentation (Issue 403)
+slider,aria,slider,slider
+aria-labelledby,aria,aria-labelledby,aria-labelledby
+aria-valuenow,aria,aria-valuenow,aria-valuenow
+aria-valuemin,aria,aria-valuemin,aria-valuemin
+aria-valuemax,aria,aria-valuemax,aria-valuemax
```

Comment:
```suggestion
aria-labelledby,specification,https://w3c.github.io/aria/#aria-labelledby,aria-labelledby
aria-valuenow,specification,https://w3c.github.io/aria/#aria-valuenow,aria-valuenow
aria-valuemin,specification,https://w3c.github.io/aria/#aria-valuemin,aria-valuemin
aria-valuemax,specification,https://w3c.github.io/aria/#aria-valuemax,aria-valuemax
```

Based on [`V2 definition spec > references.csv > values`](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#value) if I'm following this correctly

Reply:
@howard-e 

I was experimenting with a new references.csv format that is simpler for authors and creates a single source of truth for specification URLs in support.json. I've now documented the new references.csv format and updated support.json. For spec referencs that have a reference type of aria or htmlAam, the value column no longer needs to include URLs. URLs are still necessary for references of type metadata.